rules-tests/PhpSpecToPHPUnit/Rector/Variable/PhpSpecToPHPUnitRector/Fixture/fixture.php.inc

```diff
@@ -70,7 +70,7 @@ class CreateMeTest extends \PHPUnit\Framework\TestCase
 
     public function testCalled()
     {
-        /** @var spec\Rector\Tests\PhpSpecToPHPUnit\Rector\Variable\PhpSpecToPHPUnitRector\Fixture\Cart|\PHPUnit\Framework\MockObject\MockObject $cart */
+        /** @var Cart|\PHPUnit\Framework\MockObject\MockObject $cart */
```

Comment:
this is incorrect, same with property to use namespaced name, the `@var` should use namespaced as well as namespace changed.

Reply:
or possibly not, it depends on if actually the class used moved or not after namespace changed, as next `this->createMock()` uses `Cart::class`, it technically doesn't break as it doesn't add `\` prefix.

it is ok then for now.

Reply:
the outside class type actualy need to be moved under `Source` to avoid miss naming